### DISCUSSION TITLE
Migrate from Jetbrains Intellij

### DISCUSSION THREAD

**Discussion by bobaoapae** [BowlerHatLLC/vscode-as3mxml#711] [Q&A]
Currently I have a solution in intelij with multiples modules that depend in each other. Has any easy way to migrate?

**Comment by CoolBreezeArcanine** [comment]
Hey bro, I'm also working on job like this recently.

According to my tossing, it might be easier to make it work if you don't use FlexMojos.

The easiest and probable way is just create asconfig.json for each module.

But I'm not sure if it works on swc-driven project.

Feel free to show me your opinion and idea🤝
